src/services/getProductListService.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const getProductListService = async () => {
+	const res = await axios.get("/api/products");
+	if (res.status === 200) {
+		return res.data.products;
+	} else {
+		return false;
+	}
+};
```

Comment:
Making a separate file for API calls is a nice way of abstraction. Everything looks neat.